tests/hazmat/primitives/test_kbkdf.py

```diff
@@ -159,6 +159,22 @@ def test_r_type(self, backend):
                 backend=backend,
             )
 
+    def test_zero_llen(self, backend):
+        with pytest.raises(ValueError):
+            kdf = KBKDFHMAC(
```

Comment:
Ideally this should really raise in `__init__`, not wait until derive.

I think this means we want to add an extra check there, rahter than relying on the check in `int_to_bytes` (though we _also_ want that check as a backstop).